recipes/libcurl/all/conanfile.py

```diff
@@ -414,6 +435,7 @@ def _configure_cmake(self):
         # mac builds do not use cmake so don't even bother about darwin_ssl
         self._cmake.definitions["CMAKE_USE_WINSSL"] = "with_winssl" in self.options and self.options.with_winssl
         self._cmake.definitions["CMAKE_USE_OPENSSL"] = "with_openssl" in self.options and self.options.with_openssl
+        self._cmake.definitions["CMAKE_USE_WOLFSSL"] = "with_wolfssl" in self.options and self.options.with_wolfssl
```

Comment:
```suggestion
        self._cmake.definitions["CMAKE_USE_WINSSL"] = self.options.get_safe("with_winssl", False)
        self._cmake.definitions["CMAKE_USE_OPENSSL"] = self.options.get_safe("with_openssl", False)
        self._cmake.definitions["CMAKE_USE_WOLFSSL"] = self.options.get_safe("with_wolfssl", False)
```

Reply:
Current recipe doesn't conditionnally delete `with_openssl` or `with_wolfssl`.

Reply:
@Spacelm Can you explain what changes do you expect

Reply:
There is no point to check if `"with_openssl"` or `"with_wolfssl"` options exist, because they always exist.